common/beerocks/bwl/dwpal/base_wlan_hal_dwpal.cpp

```diff
@@ -146,7 +146,8 @@ bool base_wlan_hal_dwpal::fsm_setup()
                         m_dwpal_nl_ctx = nullptr;
                     } else {
                         LOG(DEBUG)
-                            << "dwpal_driver_nl_attach() success for: " << m_radio_info.iface_name;
```

Comment:
I really dislike the "before this change - after this change" commit messages. 
I propose to change the commit message to something like this:

> When an attach event is received, the dwpal_driver_nl_fd_get is not performed due to wrong condition check of the m_dwpal_nl_ctx pointer.
Fix the wrong condition check.